src/app/pages/wallets/my-wallets/wallet-form-modal/wallet-form-modal.component.spec.ts

```diff
@@ -33,4 +35,36 @@ describe('WalletFormModalComponent', () => {
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  describe('Should display error message when invalid data is provided', () => {
```

Comment:
Let's go with starting those with small letters so `should display` instead `Should display`

Reply:
Also `describe` blocks often don't contain "should" phrase. Instead they should describe some test scenario / path like:
```typescript
describe('form validation')
```